api/api/pipeline.py

```diff
@@ -36,23 +39,24 @@ def _process_img(self, img_id, session):
                 img = fr.load_image_file(fpath)
                 img_has_been_uploaded = True
                 break
-            except:
+            except Exception:
                 continue
         if img_has_been_uploaded:
             self._add_entry_to_session(Image,
                                        session,
                                        img_id=img_id)
         return img
 
+    # pylint: disable=broad-except
     def _delete_img(self, img_id):
         self.logger.debug('deleting img')
         for extension in self.allowed_file_extensions:
             img_name = "{}.{}".format(img_id, extension)
             fpath = os.path.join(self.img_dir, img_name)
             try:
                 os.remove(fpath)
-                self.logger.debug("removed {}".format(img_id))
-            except:
+                self.logger.debug("removed %s", img_id)
```

Comment:
Using `%s` here is by design since this is a logger call. If you do `self.logger.debug('removed {}'.format(img_id))`, the format string will always be allocated even if the log level is higher than debug. With `self.logger.debug('removed %s', img_id)`, the logging framework will only format the string if it's actually going to be printed.